seq2science/rules/peak_counts.smk

```diff
@@ -151,6 +151,13 @@ def get_coverage_table_replicates(file_ext):
 
     return wrapped
 
+def get_names(wildcards):
+    names = [""]
+    for rep in treps[treps["assembly"] == ori_assembly(wildcards.assembly)].index:
+        names.append(rep_to_descriptive(rep, brep=False))
+    names = "\t".join(names)
+    return names
+
```

Comment:
I double checked, this _should_ be fine. I'm not happy with the way I set this up originally to begin with. Feels like it is **very** easy to make a mistake here... :eyes:

Reply:
its the same for loop used to gather the files, so it should always be the correct order. I apply the same method to convert the RNA counts to descriptive elsewhere.

I agree its hacky, so we could leave this out this PR and come back to it later?